src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -1412,13 +1381,47 @@ int fx_muxer_t::execute(
     return result;
 }
 
-int fx_muxer_t::get_com_activation_delegate(
-    const host_startup_info_t &host_info,
-    void **delegate)
+
+static int get_delegate_from_runtime(
+    const pal::string_t& impl_dll_dir,
+    corehost_init_t* init,
+    const char* corehost_entrypoint,
+    void** delegate)
 {
-    assert(host_info.is_valid());
+    pal::dll_t corehost;
+    hostpolicy_contract host_contract{};
+    corehost_get_delegate_fn host_entry = nullptr;
```

Comment:
Can we call this with a more descriptive name? Maybe `corehost_entrypoint_function` or something like that - basically something which makes it easy to correlate to the `corehost_entrypoint` argument.